spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -101,3 +101,12 @@ fun SpdxLicense.toExpression(): SpdxLicenseIdExpression {
  */
 fun String.toSpdx(strictness: Strictness = Strictness.ALLOW_ANY): SpdxExpression =
     SpdxExpression.parse(this, strictness)
+
+/**
+ * Aggregates multiple SpdxExpressions to a [SpdxCompoundExpression] concatenated using [SpdxOperator.OR].
+ */
+fun List<SpdxExpression>.choicesToSpdx(): SpdxExpression {
+    require(this.isNotEmpty())
+
+    return this.reduceOrNull(SpdxExpression::or)!!
```

Comment:
As you already assert the list to be non-empty above, we can avoid the ugly non-null assertion here by using `reduce(SpdxExpression::or)`.

Reply:
Or we could even make it a one-liner like

    fun List<SpdxExpression>.choicesToSpdx(): SpdxExpression =
        requireNotNull(reduceOrNull(SpdxExpression::or))